extensions/amp-ad-exit/0.1/test/test-amp-ad-exit.js

```diff
@@ -757,5 +774,72 @@ describes.realWin(
         '12345'
       );
     });
+
+    it('should exit to the default target if indirect target was not set', () => {
+      const open = env.sandbox.stub(win, 'open').callsFake(() => {
+        return {name: 'fakeWin'};
+      });
+      exitIndirect();
+      expect(open).to.have.been.calledOnce;
+      expect(open).to.have.been.calledWith(
+        EXIT_CONFIG.targets.simple.finalUrl,
+        '_blank'
+      );
+    });
+
+    it('should cause error when indirect target was pointed to an invalid target', () => {
+      try {
+        allowConsoleError(() => {
+          pointTo('not-a-real-target');
+        });
+      } catch (expected) {}
+    });
```

Comment:
can this case fail? I've seen similar code of the form

try {
  foo();
} catch (e) {
  return;
}
fail();